### DISCUSSION TITLE
Permission denied when pushing to the remote repo

### DISCUSSION THREAD

**Discussion by webdevbynight** [biomejs/biome#4725] [Q&A] [answered]
Hello, everyone,

I have volunteered to translate the documentation into French.

I cloned both the biome and the website repositories. I started translating the `README.md` file of this repo into French, creating a dedicated branch. When trying to push the branch I created to the remote repo, I have the following output:
```plain
ERROR: Permission to biomejs/biome.git denied to webdevbynight.
fatal : Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

I configured the local `git remote` command for the repo as this: `git remote add origin git@github.com:biomejs/biome.git` and, when asked, I entered the passphrase of my private key without any problems.

What have I done wrong?

**Comment by ematipico** [comment, marked as answer]
Hi @webdevbynight, thank you for your interest! 

Since you don't have permissions to push **directly** to the Biome repository, [**you need to create a fork**](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo).

The page I linked you should help to set up your repository.

In few words, you push changes to your local fork, and then you send a PR to this repository. Still, all your changes are and will be only in your fork. 

Let me know if you need any more help after you read the guide, or jump on Discord

**Reply by dyc3** [reply]
@webdevbynight Personally, I recommend using the [github cli](https://cli.github.com/) to submit PRs. It automatically sets up a fork for you and does all the git remote stuff so you don't need to deal with it. If you run `gh pr create` and follow the prompts, you'll end up with a submitted PR.

**Comment by webdevbynight** [comment]
@ematipico Thank you for your answer. I forked the two repos and now two PRs are waiting for approvals. 😉
